reporter/src/main/kotlin/utils/ReportTableModelMapper.kt

```diff
@@ -135,7 +135,8 @@ class ReportTableModelMapper(
 
                 val concludedLicense = ortResult.getConcludedLicensesForId(id)
                 val declaredLicenses = ortResult.getDeclaredLicensesForId(id)
-                val detectedLicenses = licenseFindings[id]?.toSortedMap(compareBy { it.license }) ?: sortedMapOf()
+                val detectedLicenses =
+                    licenseFindings[id]?.toSortedMap(compareBy { it.license.toString() }) ?: sortedMapOf()
```

Comment:
Maybe write this as

    val detectedLicenses = licenseFindings[id].orEmpty().toSortedMap(compareBy { it.license.toString() })

to be able to keep it in one line.

Reply:
Done.